templates/flake-parts/flake.nix

```diff
@@ -7,7 +7,7 @@
       flake = false;
     };
     nixpkgs.url = "github:cachix/devenv-nixpkgs/rolling";
-    devenv.url = "github:cachix/devenv";
+    devenv.url = "github:cachix/devenv/latest";
```

Comment:
doesn't this mean that `devenv` modules won't advance any faster than cli?

Reply:
Ah, I see what you mean. I see people getting hit by bugs on `main` on first launch. Not a great experience, but I guess we need another approach here. Will revert.